test-resources/infrastructure/samconfig.toml

```diff
@@ -12,7 +12,6 @@ fail_on_empty_changeset = false
 confirm_changeset = false
 s3_prefix = "test-resources"
 region = "eu-west-2"
-resolve_s3 = true
```

Comment:
was this line causing issues?

Reply:
yes "Cannot use both --resolve-s3 and --s3-bucket parameters in non-guided deployments"